eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/http/push/AsyncHTTPPushRequest.java

```diff
@@ -350,13 +349,7 @@ ClientRetCode processResponseContent(String content) {
                     handleMsgContext.getBizSeqNo(), handleMsgContext.getUniqueId(), content);
             }
             return ClientRetCode.FAIL;
-        } catch (JsonException e) {
-            if (MESSAGE_LOGGER.isWarnEnabled()) {
-                MESSAGE_LOGGER.warn("url:{}, bizSeqno:{}, uniqueId:{},  httpResponse:{}", currPushUrl,
-                    handleMsgContext.getBizSeqNo(), handleMsgContext.getUniqueId(), content);
-            }
-            return ClientRetCode.FAIL;
-        } catch (Throwable t) {
+        } catch (Throwable e) {
```

Comment:
Suggest catch Exception and not Throwable. @Bannirui 
eg.
try {
......
} catch (JsonException e) {

} catch (Exception ex) {

}

Reply:
@jonyangx Absolutely. But i can not get the point for 2 branchs, just making a log before returning rather than some biz code... if it's better to catch `Exception` instead of `Throwable`.